opsdroid/connector/gitter/__init__.py

```diff
@@ -0,0 +1,88 @@
+
+import logging
+import aiohttp
+import asyncio
+import json
+import urllib
+
+from opsdroid.connector import Connector,register_event
+from opsdroid.events import Message
+
+_LOGGER = logging.getLogger(__name__)
+GITTER_STREAM_API       = "https://stream.gitter.im/v1/rooms"
+GITTER_MESSAGE_BASE_API = "https://api.gitter.im/v1/rooms"
+
+
+class ConnectorGitter(Connector):
+
+  def __init__(self, config, opsdroid=None):
+    """Create the connector."""
+    super().__init__(config, opsdroid=opsdroid)
+    _LOGGER.debug("Starting Gitter connector")
+    self.name = "gitter"
+    self.session = None
+    self.response = None
+    self.bot_name = self.config.get("bot-name", 'opsdroid')
+    self.room_id = self.config.get("room-id")
+    self.access_token = self.config.get("access-token")
+    self.update_interval = 1
+    self.opsdroid = opsdroid
+    self.listening = True
+
+  async def connect(self):
+    # Create connection object with chat library
+    _LOGGER.debug("Connecting with gitter stream")
+    self.session = aiohttp.ClientSession()
+    gitter_url   = self.build_url(GITTER_STREAM_API,self.room_id,"chatMessages", access_token = self.access_token)
+    self.response = await self.session.get(gitter_url,timeout=None)
+
+  def build_url(self,base_url , *res, **params):
```

Comment:
You will need to add docstrings to every function/method in order to get lint happy - this is why linting failed in a lot of places